src/key_value_db.rs

```diff
@@ -78,37 +80,32 @@ where
     #[allow(clippy::type_complexity)]
     pub(crate) fn get_changes(
```

Comment:
I don't think we ever use get_changes in madara. This function could still remain useful if it was returning `impl Iterator<Item = ExternChange>` instead of a hashmap and have the revert_to function use it instead of reimplementing its logic and be redundant.